ffi_common/src/macros.rs

```diff
@@ -56,7 +56,7 @@ the FFI boundary) so we can take care of those steps.
             #[repr(C)]
             #[allow(missing_copy_implementations)]
             #[derive(Clone, Debug)]
-            pub struct [<FFIArray $t:camel>] {
```

Comment:
Removed the case changes that use `paste` in macros. We need them to match up with the names we generate for the consumer, but we're using `heck` for that since it's String-based, and the case change implementation is slightly different between the two. It wasn't noticeable at first, but when we have something like `CLPlanting`, one will camel-case that as `CLPlanting`, and the other as `Clplanting`.

Reply:
ewww